phpstan.neon.dist

```diff
@@ -1,9 +1,13 @@
+includes:
+    - ./extension.neon
 parameters:
     level: max
     paths:
         - src
+        - tests
     reportUnmatchedIgnoredErrors: false
     checkGenericClassInNonGenericObjectType: false
+    checkMissingIterableValueType: false
```

Comment:
correct, but I didn't view is as a regression because this code isn't currently being analyzed. I can remove this line and resolve the issues if that's a blocker from merging this PR